lms/djangoapps/mobile_api/users/views.py

```diff
@@ -274,38 +234,22 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           * start: The data and time the course starts.
           * course_image: The path to the course image.
     """
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
-    permission_classes = (permissions.IsAuthenticated, IsUser)
     queryset = CourseEnrollment.objects.all()
     serializer_class = CourseEnrollmentSerializer
     lookup_field = 'username'
 
     def get_queryset(self):
-        qset = self.queryset.filter(
-            user__username=self.kwargs['username'], is_active=True
-        ).order_by('created')
-        return mobile_course_enrollments(qset, self.request.user)
+        enrollments = self.queryset.filter(user__username=self.kwargs['username'], is_active=True).order_by('created')
+        return [
+            enrollment for enrollment in enrollments
```

Comment:
Yeah, generally list comprehensions are the preferred conventional way in python, with `filter` possibly becoming deprecated.  But I could see that it could be a matter of style.
I continue to use list comprehensions for filtering since the expression makes it explicit that we're creating a list.